### PR TITLE
[DA-192] Move creating of products to UI in Product Import

### PR DESCRIPTION
**Pull request by thescouser89**
Instead of trying to create a product, we'll just assume that the
product already exists.

The REST call to the import endpoint doesn't change though. We'll still
have to send the product name.

We'll still try to create a product version if it hasn't been created
yet.

### CODE REVIEW

**Comment by thescouser89** [comment]
@jbartece I've decided to keep the REST endpoint so that you still have to send the product name as a string, instead of sending the product id, so that it is somehow still compatible with the old product import process. Let me know if you'd like me to change it to productId though! I honestly have no preference between those two (productId vs productName)

**Comment by jbartece** [comment]
@thescouser89 I'd change it as we dealt not to maintain what we don't need. So, expect only productId and do not create a product.

File: bc-backend/src/main/java/org/jboss/da/bc/backend/impl/BCSetGeneratorImpl.java [hunk 1]
```diff
@@ -41,17 +41,17 @@ public BuildConfigurationSet createBCSet(String name, Integer productVersionId,
...
 
         Product product;
 
         // create product if it does not exist
         if (!p.isPresent()) {
-            product = pnc.createProduct(new Product(name));
+            product = pnc.createProduct(new Product(productName));
         } else {
-            product = p.get();
+            throw new PNCRequestException("The product selected doesn't exist!");
```
Comment (thescouser89) [inline bc-backend/src/main/java/org/jboss/da/bc/backend/impl/BCSetGeneratorImpl.java]: @thescouser89 what the heck was that? You got the logic totally inverted.

**Comment by jbrazdil** [comment]
:+1:

**Comment by jbartece** [comment]
:+1:

**Comment by jbartece** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
Alright now I'm hitting the right testsuite failures :) Will fix the testsuite soon!

**Comment by thescouser89** [comment]
:heart:

**Comment by jbartece** [comment]
:+1:

**Pull request merged**